src/components/EditProfile/EditPersonalInformationForm.js

```diff
@@ -32,7 +32,6 @@ export default function EditPersonalInformationForm(props) {
 
     const handleChangeProfilePic = (event) => {
```

Comment:
Since this handleChangeProfilePic function is a TODO, we can remove it for now. I am also OK if you want to keep it like this.

Reply:
Sure I can remove it. I'll add the todo to this PR's description